src/react-components/room/ContentMenu.js

```diff
@@ -36,8 +36,7 @@ export function PeopleMenuButton(props) {
     <ContentMenuButton {...props}>
       <PeopleIcon />
       <span>
-        <FormattedMessage id="content-menu.people-menu-button" defaultMessage="People ({presenceCount})" values={{ presenceCount: props.presencecount }}
-        />
+        <FormattedMessage id="content-menu.people-menu-button" defaultMessage="People ({presenceCount})" values={{ presenceCount: props.presencecount }} />
```

Comment:
It seems that when the file gets formatted with prettier, it gets broken again into multiple lines due to [prettier repo settings](https://github.com/mozilla/hubs/blob/a8eb03573f4e195f7a587a2964be03081b512092/.prettierrc.json#L2)

Reply:
Huh. When does that get enforced? I'm not seeing it in VSCode, but maybe I've turned that option off. Either way I can change it to fit. This appears to be the canonical style for split-line elements:
```
      <span>
        <FormattedMessage 
          id="content-menu.people-menu-button" 
          defaultMessage="People ({presenceCount})" 
          values={{ presenceCount: props.presencecount }} 
        />
      </span>
```

Reply:
I've installed Prettier and applied it to the file. Thanks for the info.